src/main/kotlin/com/github/catppuccin/jetbrains_icons/IconProvider.kt

```diff
@@ -23,6 +23,14 @@ class IconProvider : IconProvider() {
   /** File extensions that are handled by more specific providers (not this class). */
   private val fileTypesByProviders = listOf(".java")
 
+  /**
+   * Returns an icon for the given PsiElement.
+   *
+   * @param element The PsiElement to get an icon for.
+   * @param flags Additional flags for icon retrieval (not used in this implementation).
+   * @return The icon for the element, or null if no suitable icon is found or if the file type is
```

Comment:
Do not start comments like `@return` and `@param` with capitals. Also this is repeated in the main description